dasharo-security/secure-boot.robot

```diff
@@ -285,6 +285,58 @@ SBO009.002 Check automatic certificate provisioning KEK certificate
     ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
     Reset To Default Secure Boot Keys    ${advanced_menu}
 
+SBO010.001 Enroll certificates using sbctl
+    [Documentation]    This test installs sbctl and verifies that it is possible
+    ...    to use it to enroll certificates.
+    Skip If    not ${SECURE_BOOT_SUPPORT}    SBO010.001 not supported
+    Power On
+    # Erase Secure Boot Keys
+    ${sb_menu}=    Enter Secure Boot Menu And Return Construction
+    Enable Secure Boot    ${sb_menu}
+    Save Changes
+    Reenter Menu
+    ${sb_menu}=    Get Secure Boot Menu Construction
+    ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
+    Erase All Secure Boot Keys    ${advanced_menu}
+    Save Changes And Reset    3    5
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    # Install sbctl
+    Execute Linux Command    apt install -y git util-linux binutils golang-go asciidoc make    120
+    Execute Linux Command    [ ! -d "sbctl" ] && git clone https://github.com/foxboron/sbctl.git
+    ${new_prompt}=    Get Linux Prompt In Directory    sbctl    ${DEVICE_UBUNTU_ROOT_PROMPT}
+    Set Prompt For Terminal    ${new_prompt}
+    Execute Linux Command    cd sbctl
+    Execute Linux Command    make    50
+
+    # Create and enroll keys
+    Execute Linux Command    ./sbctl create-keys
+    ${out}=    Execute Linux Command    ./sbctl enroll-keys --yes-this-might-brick-my-machine
```

Comment:
@PLangowski I think that even better it would be to use keywords for those commands like here https://github.com/Dasharo/open-source-firmware-validation/pull/161/files#diff-b06ff90d8f021dfa2e18d217f3d4c1e6861d6f671893cf292b6718b80227921aR222

Reply:
Amended here 8a333e00f2f8904c7c2281796c840b299cd991c9.